product_supplierinfo_disable_autocreation/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Juan Alberto Raja <juan.raja@sygel.es>
+License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record
+        id="vendor_pricelist_disable_res_config_settings_view_form"
```

Comment:
Could you modify the id of the view with the new module name? Make sure that the translations are also modified.

Reply:
@BetoRaja ready?